documentation/modules/tracing/proc-configuring-jaeger-tracer-kafka-clients.adoc

```diff
@@ -21,23 +25,50 @@ In each client application:
     <version>{JaegerClientVersion}</version>
 </dependency>
 ----
++
+.Dependencies for Jaeger tracing with OpenTelemetry
+[source,xml,subs="attributes+"]
+----
+<dependency>
+    <groupId>io.opentelemetry</groupId>
+    <artifactId>opentelemetry-sdk-extension-autoconfigure</artifactId>
+    <version>{OpenTelemetryVersion}</version>
+</dependency>
+<dependency>
+    <groupId>io.opentelemetry</groupId>
+    <artifactId>opentelemetry-exporter-jaeger</artifactId>
+    <version>{OpenTelemetryVersionStable}</version>
+</dependency>
+<dependency>
+    <groupId>io.grpc</groupId>
+    <artifactId>grpc-netty-shaded</artifactId>
+    <version>{OpenTelemetryGrpcVersion}</version>
+</dependency>
```

Comment:
Tbh I have Kafka clients instrumented without the need for this grpc dependency.